modules/gdscript/gdscript_warning.cpp

```diff
@@ -123,7 +123,7 @@ String GDScriptWarning::get_message() const {
 		} break;
 		case UNSAFE_CALL_ARGUMENT: {
 			CHECK_SYMBOLS(4);
-			return "The argument '" + symbols[0] + "' of the function '" + symbols[1] + "' requires a the subtype '" + symbols[2] + "' but the supertype '" + symbols[3] + "' was provided";
+			return vformat(R"*(The argument %s of the function "%s()" requires a the subtype "%s" but the supertype "%s" was provided.)*", symbols[0], symbols[1], symbols[2], symbols[3]);
```

Comment:
There's a typo in `requires a the subtype` :)

Reply:
Okay, I'll fix this. :)

Reply:
Done.